src/shared/sandboxCommandBase.ts

```diff
@@ -50,9 +51,29 @@ export abstract class SandboxCommandBase<T> extends SfCommand<T> {
     return this.sandboxRequestConfig;
   }
 
+  protected async calculateTrackingSetting(tracking = true): Promise<boolean> {
```

Comment:
core has something similar to handle orgs where the `tracksSource` property doesn't exist, but it's more expensive (trying to figure out what type of org it is).  This one is better because 1) we know it's a sbox 2) we know the license type.